picard/ui/__init__.py

```diff
@@ -27,3 +28,16 @@ class PicardDialog(QtWidgets.QDialog):
 
     def __init__(self, parent=None):
         QtWidgets.QDialog.__init__(self, parent, self.flags)
+
+
+class HashableTreeWidgetItem(QtWidgets.QTreeWidgetItem):
```

Comment:
This commit and 9ffc73b0bb6facc0895f3dd73508abae196b5e40 are really in need of an explanation: neither the class documentation nor the commit message explain why this is necessary. I guess it's because of [this](https://stackoverflow.com/questions/42140366/qtreewidgetitem-not-hashable-in-python3)?